### DISCUSSION TITLE
Welcome to ViPER4Android Presets Discussions!

### DISCUSSION THREAD

**Discussion by syntaxticsugr** [syntaxticsugr/ViPER4AndroidFXRepack-Presets#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by WSTxda** [comment]
Hi, I am currently a developer of [ViperFX RE](https://github.com/WSTxda/ViperFX-RE-Releases), I would like to integrate this repo into the audio files download options.

![Screenshot_2024-10-16-08-47-01-242_com.wstxda.viper4android-edit.jpg](https://github.com/user-attachments/assets/b5a21879-58d3-4a76-9a67-5af67168f69c)

![Screenshot_2024-10-16-08-46-18-983_com.wstxda.viper4android-edit.jpg](https://github.com/user-attachments/assets/685e91e6-7bee-4a41-80ea-ae7a1f7b0e4b)

I believe that this repository is currently the most complete and it would be simpler to use.

Tell me what you think about this. Thanks! :)

**Reply by WSTxda** [reply]
@syntaxticsugr tell me what you think

**Reply by syntaxticsugr** [reply]
Hi @WSTxda , Thankyou for reaching out and for your contributions to ViPER.
I’m totally on board with "integrating my repo into your audio files download option". It sounds like a great way to make it easier for ViPER users to download and use the presets. Let me know if I can assist in any way possible with the process.
